gpswearos/src/main/java/com/android/gpswearos/presentation/SampleData.kt

```diff
@@ -0,0 +1,15 @@
+package com.android.gpswearos.presentation
+
+object SampleData {
+    // Sample conversation data
+    val statusSample = listOf(
+        SatelliteStatus("Lat:", "28.0713003 °"),
+        SatelliteStatus("Time:", "04:35:29 PM"),

```

Comment:
Let's see if we can remove time from the content list and instead populate the `timeText` in the Scaffold with the GPS time.

Reply:
> Let's see if we can remove time from the content list and instead populate the `timeText` in the Scaffold with the GPS time.

Ok, I've replaced it.